test/unit/winston/rejection-handler.test.js

```diff
@@ -69,12 +69,14 @@ describe('UnhandledRejectionHandler', function () {
   });
 
   it('.handle()', function (done) {
+    process.removeAllListeners('unhandledRejection');
```

Comment:
This is the line in question.  If possible, it's probably best to make a copy of mocha's listener, then remove all, then run the test, then remove all again at the end of the test (i.e., ensure the one we add during the test is removed) and finally re-add mocha's.